src/components/feedback/FeedbackList.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { FC } from 'react';
+import { useTheme } from '@emotion/react';
+
+import FeedbackItem from './FeedbackItem';
+import { feedbackListWrapperStyle, feedbackTotalCountStyle } from './style';
+
+import type { Feedback } from '@/@types/project';
+
+type FeedbackListProps = {
+  feedbacks: Feedback[];
+  totalFeedBackCount: number;
+  primary: boolean;
+};
+
+const FeedbackList: FC<FeedbackListProps> = ({
+  feedbacks,
+  primary,
+  totalFeedBackCount,
+}) => {
+  const theme = useTheme();
+  return (
+    <div>
+      <ul
+        css={feedbackListWrapperStyle}
+        style={feedbacks.length <= 2 ? { justifyContent: 'center' } : {}}
+      >
+        {feedbacks.map(({ id, title, contents, userName }) => {
+          return (
+            <FeedbackItem
+              key={`feedback-${id}`}
+              title={title}
+              contents={contents}
+              userName={userName}
+            />
+          );
+        })}
+      </ul>
+      <p
+        css={[
+          feedbackTotalCountStyle,
+          { color: primary ? theme.colorSubLight : theme.colorSuccess },
+        ]}
+      >
```

Comment:
요기도 prop으로 처리해도 좋을 것 같아요.

```suggestion
      <p
        css={feedbackTotalCountStyle(primary)}
      >
```

Reply:
이 부분 혹시 테마 그 부분 아닌가욥??

Reply:
맞네용!...

Reply:
이건 아마 수정 되었을 겁니다 다른 PR에서..